tests/experimental/pydantic/test_conversion.py

```diff
@@ -55,6 +57,68 @@ class UserType:
     assert user.password == "abc"
 
 
+def test_cannot_convert_pydantic_type_to_strawberry_missing_field():
+    class User(BaseModel):
+        age: int
+
+    with pytest.raises(
+        AutoFieldsNotInBaseModelError,
+        match=re.escape(
+            "UserType defines ['password'] with strawberry.auto."
+            " Field(s) not present in User BaseModel."
+        ),
+    ):
+
+        @strawberry.experimental.pydantic.type(User)
+        class UserType:
+            age: strawberry.auto
+            password: strawberry.auto
+
+
+def test_cannot_convert_pydantic_type_to_strawberry_property_auto():
+    # auto inferring type of a property is not supported
```

Comment:
should this be supported in future? 🤔

Reply:
that would be nice, though idk how complicated it would be. i think we could isinstance(method, property) to check if its a property. and then we could extract the return type from the __code__.